packages/core/src/utilities/VoxelValue.ts

```diff
@@ -0,0 +1,235 @@
+import type { BoundsIJK, Point3, VolumeScalarData } from '../types';
+
+/**
+ * This is a simple, standard interface to values associated with a voxel.
+ */
+export default class VoxelValue<T> {
```

Comment:
I feel like this is a VoxelManager rather than VoxelValue, since if you take a look at the way we use it (e.g., getStrategyData) it is like

```js
return {
    segmentationImageData,
    segmentationScalarData,
    segmentationVoxelValue: VoxelValue.volumeVoxelValue(
      dimensions,
      segmentationScalarData
    ),
    imageScalarData,
    imageVoxelValue: VoxelValue.volumeVoxelValue(dimensions, imageScalarData),
  };
  ```

So here `segmentationVoxelValue` sounds like it is just a value (one scalar), but it is not, it is set of utilities 

what do you think